pyiron_atomistics/sphinx/base.py

```diff
@@ -628,9 +622,6 @@ def calc_static(
 
         Args:
             electronic_steps (float): max # of electronic steps
-            retain_electrostatic_potential:
-            retain_charge_density:
-            algorithm (str): CCG or blockCCG (not implemented)
             electronic_steps (int): maximum number of electronic steps
                 which can be used to achieve convergence
         """
```

Comment:
What is this if statement doing? Is electronic steps ever actually None?

Reply:
Which if statement?

Reply:
the one right below, sorry, vscode won't let me select it

Reply:
ah yeah it's kind of complicated (actually super sketchy) because in `interactive.py` I guess it's `None` by default, which makes it extremely difficult to understand. I'm ready to change it but since it's not really related to this PR I might want to open another one if you think it should be changed.